homeassistant/helpers/service.py

```diff
@@ -58,6 +60,67 @@
 SERVICE_DESCRIPTION_CACHE = "service_description_cache"
 
 
+def validate_attribute_option(attribute_option: str) -> Any:
+    """Validate attribute option."""
+    domain, enum, option = attribute_option.split(".", 3)
+
+    integration = importlib.import_module(f"homeassistant.components.{domain}")
```

Comment:
Why not re-use the same logic from `selector.py` ?

Reply:
I like that logic more than importing based on a string. That feels too fragile.

Reply:
The reason it's done like that in selector.py is that we shouldn't import modules when validating.

I don't agree it's fragile, we validate all service descriptions in `hassfest` which will give meaningful errors if resolving the enum value fails.

Reply:
do we validate custom components?

Reply:
Yes, at runtime.
Anyway, modified a bit according to our discussion.